src/data/finset/pointwise.lean

```diff
@@ -837,4 +842,10 @@ localized "attribute [instance] finset.distrib_mul_action_finset
   finset.mul_distrib_mul_action_finset" in pointwise
 
 end instances
+
+lemma pairwise_disjoint_smul_iff [decidable_eq α] [left_cancel_semigroup α] {s t : finset α} :
```

Comment:
In this lemma, the finset `s` only appears through its coercion to `s : set α`. It would probably be a better idea to spell the lemma with a general `s : set α` directly.

Reply:
Ah, but not `t`?